jcli/Cargo.toml

```diff
@@ -24,12 +24,12 @@ bech32 = "0.8"
 hex = "0.4.2"
 rayon = "1.5"
 base64 = "0.13.0"
-chain-core      = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "catalyst-fund7" }
-chain-impl-mockchain = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "catalyst-fund7" }
-chain-addr      = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "catalyst-fund7" }
-chain-crypto    = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "catalyst-fund7" }
-chain-time    = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "catalyst-fund7" }
-chain-vote = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "catalyst-fund7" }
+chain-core      = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "update-proposal-certificate" }
+chain-impl-mockchain = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "update-proposal-certificate" }
+chain-addr      = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "update-proposal-certificate" }
+chain-crypto    = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "update-proposal-certificate" }
+chain-time    = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "update-proposal-certificate" }
+chain-vote = { git = "https://github.com/input-output-hk/chain-libs.git", branch = "update-proposal-certificate" }
```

Comment:
I don't think this should go in

Reply:
Of course, I have recently updated PR with the latest version from the fund7 branch :)